dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/AnnotationScanner.java

```diff
@@ -166,19 +167,31 @@ private String findServletMappingForApp(String appClassName) throws UnableToAdap
         return getUrlMappingFromApp(appClassName);
     }
 
+    @FFDCIgnore(UnableToAdaptException.class)
+    public synchronized Set<String> getAnnotatedClassesNames() {
+        AnnotationTargets_Targets annotationTargets;
+        Set<String> restAPIClasses = null;
+
+        try {
+            annotationTargets = webAnnotations.getAnnotationTargets();
+            restAPIClasses = ANNOTATION_CLASS_NAMES.stream().flatMap(anno -> annotationTargets.getAnnotatedClasses(anno,
+                                                                                                                   AnnotationTargets_Targets.POLICY_SEED).stream()).collect(Collectors.toSet());
+            Tr.event(tc, "Found annotated classes: ", restAPIClasses);
+
+        } catch (UnableToAdaptException e) {
+            e.printStackTrace();
```

Comment:
Should this be changed to Tr.debug / Tr.event statement?

Reply:
Thanks, added Tr.event instead

Reply:
looks good. thanks.